nexus/src/app/sagas/instance_create.rs

```diff
@@ -179,7 +174,7 @@ impl NexusSaga for SagaInstanceCreate {
         // Helper function for appending subsagas to our parent saga.
         fn subsaga_append<S: Serialize>(
             node_basename: &'static str,
-            subsaga_builder: steno::DagBuilder,
+            subsaga_dag: steno::Dag,
```

Comment:
On @andrewjstone's recommendation I updated this function to take a dag instead of a builder. It's easy enough to call the builder at the append callsite and this lets us reuse the disk create saga's `make_saga_dag` function.